### PR TITLE
[r] Test-drive `apis/r/README.md` on a fresh EC2 instance

### PR DESCRIPTION
**Pull request by johnkerl**
**Issue and/or context:** More on issue #1032 following PR #1054

**Changes:** Results of a a test-drive on a from-scratch host

### CODE REVIEW

File: apis/r/README.md [hunk 1]
```diff
@@ -29,10 +29,11 @@ $ install.packages('tiledbsoma', repos = c('https://tiledb-inc.r-universe.dev',
...
 * Clone this repository: `git clone https://github.com/single-cell-data/TileDB-SOMA.git`
 * Change into the R API package directory: `cd TileDB-SOMA/apis/r`
-* Optionally, clean the file in to the repo: `./cleanup` (this is not needed the first time)
+* Optionally, clean the files in the repo: `./cleanup` (this is not needed the first time)
 * Optionally, update the `libtiledbsoma` sources: `./copy_source.sh` (which updates the includes tarball of `libtiledbsoma`).
-* Build the R package source tarball from the repository sources: `R CMD build .` (which will also build `libtiledbsoma` from source; other dependencies are required as described in the previous section).
-* Check and test the package from the tarball: `R CMD check tiledbsoma_*.tar.gz`
+* Build the R package source tarball from the repository sources: `R CMD build .` (which will also build `libtiledbsoma` from source; other dependencies are required as described in the previous section)
+* Check and test the package from the tarball: `R CMD check --no-vignettes --no-manual tiledbsoma_*.tar.gz`
+  * For quicker iteration, after `Rscript -e 'install.packages("pkgbuild")'`, run `Rscript -e 'testthat::test_local(".")'`
```
Comment (eddelbuettel) [inline apis/r/README.md:36]: I suspect we can skip `install.packages("pkgbuild")` here. It is no suggested by `testthat` (whereas `pkgload` is already dependency -- and used in the code.  Typo on your end?

Also, I think from the package directory we need `testthat::test_local("tests/testthat")`

**Review by eddelbuettel — COMMENTED** [review]
See comment in line.

File: apis/r/README.md (hunk 1 above)
Comment (johnkerl) [reply apis/r/README.md:36]: oops yes of course thank you!!

**Comment by johnkerl** [comment]
@eddelbuettel typo fixed! :)

File: apis/r/README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ $ install.packages('tiledbsoma', repos = c('https://tiledb-inc.r-universe.dev',
 
 * Source installation requires the [`tiledb` R package](https://github.com/TileDB-Inc/TileDB-R) (which in turn depends on the [`tiledb` Core library](https://github.com/TileDB-Inc/TileDB)).
 * In general, source installation of TileDB and its packages may require `cmake` and `git` to be installed; these are common tools each operating system provides readily.
-* Other R package dependencies, are listed in the [DESCRIPTION](https://github.com/single-cell-data/TileDB-SOMA/blob/main/apis/r/DESCRIPTION) file and can be installed via _e.g_
+* Other R package dependencies, are listed in the [DESCRIPTION](https://github.com/single-cell-data/TileDB-SOMA/blob/main/apis/r/DESCRIPTION) file and can be installed via _e.g._
   `remotes::install_deps(".")`. In order build vignettes, `knitr` and `rmarkdown` are required as is `testthat` for testing.
```
Comment (eddelbuettel) [inline apis/r/README.md:24]: Add 'to' to yield "In order to build vignettes"

Comment (eddelbuettel) [inline apis/r/README.md:24]: `install_deps()` needs the second argument to install the data package so please change to

     install.deps(".", TRUE)

**Review by eddelbuettel — CHANGES_REQUESTED** [review]
See inline

**Review by eddelbuettel — CHANGES_REQUESTED** [review]
Looks good -- one micro nit.

**Review by eddelbuettel — APPROVED** [review]
:shipit:

**Pull request merged**